mqtt/mqtt-broker/src/broker.rs

```diff
@@ -79,19 +79,22 @@ where
                                 info!("sent state to snapshotter.");
                             }
                         }
-                        SystemEvent::ForceClientDisconnect(client_id) => {
```

Comment:
is this no longer needed?

Reply:
Nope, we can re-use `ClientEvent::DropConnection` which has the same semantics